### DISCUSSION TITLE
Change to flet 0.23

### DISCUSSION THREAD

**Discussion by ingdesarr1** [flet-dev/flet#3802] [Q&A] [answered]
### Question

Good afternoon. I currently have an application developed in flet 0.21.2, where many of the elements to use are usercontrol, for ease of access, compacting some elements and so on. I made a change to the current version, 0.23.2, and many messages appear about usercontrols being deprecated, and many other functions. Is there any other element that replaces the usercontrol or how can I migrate for future versions?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
As you might have noticed, the deprecation message contains a link to the documentation, on how to build custom controls: https://flet.dev/docs/getting-started/custom-controls
